src/rp2_common/pico_atomic/include/stdatomic.h

```diff
@@ -0,0 +1,20 @@
+#ifndef __STDATOMIC_H
+#define __STDATOMIC_H
+
+#include_next <stdatomic.h>
```

Comment:
phew; i was wondering what versions of compilers supported this, but GCC 6 is happy, so OK  i guess!

Reply:
I could drop a complete replacement for stdatomic.h as different approach?  Seem like that would be worse in the long term.

Reply:
yeah; i am happy with `include_next` i just thought it was a recent add